### DISCUSSION TITLE
Kata's should be graded

### DISCUSSION THREAD

**Discussion by shakurah** [codewars/codewars.com#2605] [Ideas]
Instead of just showing you passed katas should be graded on
-How well your code performed
-How well written it is(use of functions and objects)

**Comment by hobovsky** [comment]
Measured of performance is not planned for now. You can see https://github.com/codewars/codewars.com/issues/959 and linked issues for some details why.

Code style would be quite difficult to be graded automatically.

**Reply by shakurah** [reply]
Ohk thanks
